xiblint/rules/color_assets.py

```diff
@@ -0,0 +1,80 @@
+from xiblint.rules import Rule
+from xiblint.xibcontext import XibContext
+import glob
+import json
+
+class ColorAssets(Rule):
+    def __init__(self, config):
+        asset_catalog_path = config.get('asset_catalog', None)
+        if asset_catalog_path is None:
+            # TODO: Proper error
+            print("Asset catalog not found. Please configure 'asset_catalog'.")
```

Comment:
hrm, I'm not sure what the right way to error is, but maybe you can:

```suggestion
            sys.exit("error: Asset catalog not found. Please configure 'asset_catalog'.")
```

Reply:
`raise SystemExit('error: ...')` slightly more idiomatic